apps/users/forms.py

```diff
@@ -95,20 +95,3 @@ def __init__(self, key, user, metadata=None, **kwargs):
             "class": "form-control",
             "placeholder": "Enter token from authenticator app...",
         }
-
-    def clean_token(self):
```

Comment:
The `clean_token` method is included within the parent class and gives all of the functionality required to clean the inpiut. This is therefore superfluous.